app/helpers/sufia/sufia_helper_behavior.rb

```diff
@@ -121,6 +121,11 @@ def iconify_auto_link(text, showLink = true)
       end
     end
 
+    def iconify_link(text, showLink = true)
```

Comment:
Does this need to take a second argument?  Seems like it would only be used where showLink is false.  Also, variables should be snake case instead of camel case.